### PR TITLE
micromamba 1.3.1

### PR DESCRIPTION
**Pull request by pavelzw**
~~Created with `brew bump-formula-pr`.~~ Modified afterwards

`resource` blocks may require updates.

### CODE REVIEW

**Review by carlocab — DISMISSED** [review]
I think this needs a `livecheck` block that converts their date-tags into the associated version. CC @samford

**Comment by pavelzw** [comment]
We could also use the `micromamba-1.x.x` tags... This is also done in the official micromamba distribution on conda-forge, see [here](https://github.com/conda-forge/micromamba-feedstock/blob/543fc351ef43a86d8d488f1edd51e0a51a54fec7/recipe/meta.yaml#L9).

The `micromamba-1.x.x` tag *should* point to the same tag as the date tag.

**Comment by carlocab** [comment]
Hmm, yea, might be nicer. We'll want to exclude the date tags from `livecheck` still though, since:
```
❯ brew livecheck micromamba
micromamba: 1.3.0 ==> 2023.02.09
```

**Comment by pavelzw** [comment]
```rb
livecheck do
  url :stable
  strategy :git do |tags|
    tags.map { |tag| tag[(/^micromamba-(\d+(?:\.\d+)+)$/i)]&.gsub(/micromamba-/, "") }
  end
end
```

This seems to work... Like this, we ignore all unnecessary tags (i.e. `mamba-1.x.x`, `libmamba-1.x.x`, `20xx.xx.xx`). But I'm not really proficient with ruby, if there is a better version to do it please let me know.

File: Formula/micromamba.rb [hunk 1]
```diff
@@ -1,11 +1,17 @@
...
   desc "Fast Cross-Platform Package Manager"
   homepage "https://github.com/mamba-org/mamba"
-  url "https://github.com/mamba-org/mamba/archive/refs/tags/2023.02.03.tar.gz"
-  version "1.3.0"
-  sha256 "4c794246dd6008caebebef8c6266e6ba3f9ff5ce48908c830e448811587e9921"
+  url "https://github.com/mamba-org/mamba/archive/refs/tags/micromamba-1.3.1.tar.gz"
+  sha256 "af66b79bc6b9673209e92b6ce6e52a21a79c55e86960a9d5bb10691cf2b4e327"
   license "BSD-3-Clause"
 
+  livecheck do
```
Comment (samford) [inline Formula/micromamba.rb:8]: I can't leave a proper suggestion from the mobile website but this `strategy` block is unnecessary and this should simply use `regex(/^micromamba[._-]v?(\d+(?:\.\d+)+)$/i)` instead.

Comment (pavelzw) [reply Formula/micromamba.rb:8]: Thanks!

**Review by carlocab — APPROVED** [review]

**Pull request closed without merging**